src/components/editor/GroupChoiceModal.jsx

```diff
@@ -102,15 +93,8 @@ const mapDispatchToProps = dispatch => ({
   closeRdfPreview() {
     dispatch(showRdfPreview(false))
   },
-  setBaseURL(url) {
-    dispatch(assignBaseURL(url))
-    dispatch(showResourceURIMessage(url))
-  },
-  saveStarted() {
-    dispatch(updateStarted())
-  },
-  saveFinished(rdf) {
-    dispatch(updateFinished(generateMD5(rdf)))
+  publishMyResource: (user, group) => {
+    dispatch(publishResource(user, group))
```

Comment:
is this needed, or could i just call publishResource() directly in saveAndClose() ?  It seems like a silly wrapper.

Reply:
Needed because you need `dispatch`.  That is not available in saveAndClose.